langchain-llamaindex-slackbot/src/slackbot_utils.py

```diff
@@ -58,7 +58,8 @@ def connect_to_zenml_server():
 
 def get_vector_store():
     """Get the vector store from the pipeline."""
-    pipeline = get_pipeline(pipeline=PIPELINE_NAME, version=9)
+    # gets the most recent pipeline version
+    pipeline = get_pipeline(pipeline=PIPELINE_NAME).versions[0]
     our_run = pipeline.runs[0]
     print("Using pipeline: ", pipeline.model.name, "v", pipeline.model.version)
     print("Created on: ", pipeline.model.updated)
```

Comment:
Version accessing is really confusing, I think versions are sorted by creation time, not by time of their last runs, so `versions[0]` might not always work as expected. However, `pipeline.runs[0]` should always give you the last run under that pipeline name at least.

```suggestion
    pipeline = get_pipeline(pipeline=PIPELINE_NAME)
    our_run = pipeline.runs[0]
    print("Using pipeline: ", pipeline.name)
```

Reply:
In testing, this actually worked, but your way seems less confusing so let's go with that. In any case it's an improvement on manually specifying the version (which we were previously doing).